README.md

```diff
@@ -14,33 +14,67 @@ First, install [ChefDK](https://downloads.chef.io/chef-dk/), then:
     $ kitchen converge  # provisions/installs from chef cookbooks
     $ kitchen login     # ssh in to vm
 
+### MessyBrainz and ListenStore
+
+ListenStore is our interface to cassandra. It's bundled as part of this
+repository, but you must install it so that python can find it.
+
+    $ cd listenstore
+    $ python setup.py develop
+
+You can use `setup.py develop` to symlink the listenstore directory into
+site-packages (letting you make changes easily), or use `setup.py install`
+to install it. If you make any changes to listenstore you must install
+again.
+
+MessyBrainz is an intermediate database to map unknown metadata to stable
+UUIDs, and then map these intermediate identifiers to MBIDs
+(http://messybrainz.org, https://github.com/metabrainz/messybrainz-server)
```

Comment:
I don't think this is necessary. It looks just fine here: https://github.com/metabrainz/listenbrainz-server/blob/c9aad/README.md#messybrainz-and-listenstore. GitHub is adding link tags implicitly.